config/application.rb

```diff
@@ -34,5 +34,8 @@ class Application < Rails::Application
       g.fixture_replacement :factory_girl, :dir => 'spec/factories'
     end
     config.autoload_paths << Rails.root.join('lib')
+    config
+      .browserify_rails
+      .commandline_options = "-t [ reactify --es6 target --es5 ]"
```

Comment:
Place the . on the previous line, together with the method call receiver.<br>Prefer single-quoted strings when you don't need string interpolation or special symbols.